functional-tests/mixins/bridge_mixin.py

```diff
@@ -181,13 +181,16 @@ def make_drt(self, el_address, musig_bridge_pk):
         # Send the transaction to the Bitcoin network
         drt_tx_id: str = self.btcrpc.proxy.sendrawtransaction(tx)
 
+        # FIXME: what are we waiting for here?
         time.sleep(1)
 
         # time to mature DRT
         self.btcrpc.proxy.generatetoaddress(6, seq_addr)
+        # FIXME: what are we waiting for here?
         time.sleep(3)
 
         # time to mature DT
         self.btcrpc.proxy.generatetoaddress(6, seq_addr)
+        # FIXME: what are we waiting for here?
```

Comment:
Not sure what the waiting condition should be here.

Reply:
Transactions being included in blocks.  Need some way to wait on that.